api/src/extensions/lib/get-extensions.ts

```diff
@@ -1,54 +1,25 @@
 import { useEnv } from '@directus/env';
-import type { Extension } from '@directus/extensions';
-import { getLocalExtensions, getPackageExtensions, resolvePackageExtensions } from '@directus/extensions/node';
+import { resolveDependencyExtensions, resolveExtensions } from '@directus/extensions/node';
 import { useLogger } from '../../logger.js';
 import { getExtensionsPath } from './get-extensions-path.js';
 
 export const getExtensions = async () => {
 	const env = useEnv();
 	const logger = useLogger();
+	const extensions = await resolveExtensions(getExtensionsPath());
```

Comment:
Would prefer if we use the term "local extensions" to make it clearer and also consistent with `vite.config.js`. 
```suggestion
	const localExtensions = await resolveLocalExtensions(getExtensionsPath());
```

Reply:
Deal 🤝